create_a_materialized_view_batch_ingest/deploy/0.0.1/cd-deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+tb deploy --populate --wait
```

Comment:
ci-deploy.sh and  cd-deploy.sh have been unified to deploy.sh. But for the populate we recommend to use "post_deploy.sh" and do it in a regular populate operation. My recommendation is to leave the default deploy (not to add deploy.sh) and run a pipe populate in the post_deployment.sh.